tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/locales/scriptToExemplars.txt

```diff
@@ -0,0 +1,26 @@
+# Contains sample aux exemplars for a hint in examples, for use in ScriptToExemplars.java
+# It is generated from the different sets of exemplars that share the same script.
+#   Eg, if locale1 and locale2 use the same script, and locale1 has exemplars [abcd] and locale2 has [cdef], then the set will be [af]
```

Comment:
perhaps: ... set will be [a-f]

Reply:
It leaves out all the common characters to all the locales sharing the script, since they won't be added.